humble/distribution.yaml

```diff
@@ -5454,6 +5454,24 @@ repositories:
       url: https://github.com/ros-visualization/python_qt_binding.git
       version: humble
     status: maintained
+  qbshin-ros:
+    release:
```

Comment:
Are you interested in adding a `doc` entry?

Reply:
No, maybe we will add it in the next version

Reply:
OK.  Just so you know, adding a `doc` entry means that our buildfarm will automatically attempt to generate API documentation for your package, and also register it so users can find it in the ROS index.

But it is not at all required, so if you don't want to add it here that's fine.